.eslintrc.js

```diff
@@ -17,7 +17,8 @@ module.exports = {
     'jest/no-focused-tests': 'error',
     'jest/no-identical-title': 'error',
     'jest/prefer-to-have-length': 'warn',
-    'jest/valid-expect': 'error'
+    'jest/valid-expect': 'error',
+    'indent': 'off'
```

Comment:
Why did you remove this? Indentation should follow a strict scheme.

Reply:
We only have added the `'indent': 'off'`, if not set the compilation hangs.
But it's perhaps a thing to keep in mind! 
Perhaps specific to our Dev config? Don't known if we should have keep it in the final commit.

Reply:
I removed it. The compilation hangs because it waits that you fix the indentation of the mentioned files. It's here to make sure the code stays similar everywhere :).